api/constants/constants.go

```diff
@@ -134,3 +134,8 @@ const (
 	// requests a connection to the remote auth server.
 	RemoteAuthServer = "@remote-auth-server"
 )
+
+const (
+	// TODO(Joerger): change this to generated value
+	Version = "7.0.0-dev"
```

Comment:
We already have version in `version.go` file. Is this in addition to that? Does this mean we'd need to bump version in 3 places now when doing releases (Makefile, version.go and here)?

Reply:
`teleport/api/` doesn't depend on `teleport/` which is why I can't really use the value from `version.go`. It sounds like @Joerger is working on something so that `api/` will have access to the generated client version, the suggestion was to copy this here for now and let him fill in the generated value when his changes come in. I'm not sure how quickly those changes are coming or if we should put in something to get the generated value for this PR @Joerger ? fyi I think we want this going out in the 6.2 train

Reply:
Here's the PR for the client version generation - #7157